dataflow/src/main/java/org/checkerframework/dataflow/expression/Unknown.java

```diff
@@ -31,7 +67,7 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return "?";
+        return originalExpression;
```

Comment:
We could add strings to `originalExpression` to make it clear that the expression is unknown.  Something like `"Unknown(" + originalEI + ")"`. don't know if that's necessary.